src/main/java/teammates/logic/api/Logic.java

```diff
@@ -111,6 +126,13 @@ public String getCourseInstitute(String courseId) {
         return notificationsLogic.getAllNotifications();
     }
 
+    /**
+     * Checks if a notification of {@code notificationId} exists.
+     */
+    public boolean doesNotificationExists(String notificationId) {
```

Comment:
Checking if a notification exists can be achieved by calling `getNotification` and checking if the result is null, so this extra function (and the corresponding functions in `NotificationsLogic` and `NotificationsDb`) may not be necessary.

Reply:
Good point, but take note that depending on the situation, in some cases if only checking for existence is needed i.e. no need the actual object itself, a method like `doesNotificationExists` which can perform a key-only query can be beneficial in terms of performance/reducing reads.

Reply:
Yes the method checks using key only query, which saves cost for reading